src/tests/Microsoft.PowerFx.Interpreter.Tests/MutationScripts/User_SealedRecord.txt

```diff
@@ -0,0 +1,110 @@
+#SETUP: PowerFxV1CompatibilityRules
+
+// User and table of User works as normal
+
+>> User
+{DataverseUserId:GUID("aa1d4f65-044f-4928-a95f-30d4c8ebf118"),Email:"susan@contoso.com",FullName:"Susan Burk",TeamsMemberId:"29:1DUjC5z4ttsBQa0fX2O7B0IDu30R"}
+
+>> [User]
+Table({DataverseUserId:GUID("aa1d4f65-044f-4928-a95f-30d4c8ebf118"),Email:"susan@contoso.com",FullName:"Susan Burk",TeamsMemberId:"29:1DUjC5z4ttsBQa0fX2O7B0IDu30R"})
+
+// AddColumns and RenameColumns are blocked
+
+>> AddColumns( User, Foo, 1 )
+Errors: Error 12-16: Can't add columns to a system record or table of system records. Use ShowColumns to extract the specific columns you need first.|Error 0-26: The function 'AddColumns' has some invalid arguments.
+
+>> RenameColumns( User, FullName, Name )
+Errors: Error 15-19: Can't rename columns of a system record or table of system records. Use ShowColumns to extract the specific columns you need first.|Error 0-37: The function 'RenameColumns' has some invalid arguments.
+
+// DropColumns and ShowColumns work as normal, as they are removing without adding
+
+>> DropColumns( User, FullName )
```

Comment:
If the input of DropColumns is locked, we should also lock the result, so that. This way one cannot add a new column to a locked type.

Reply:
Nevermind, saw that this is already done.